src/black/__init__.py

```diff
@@ -1361,9 +1264,50 @@ def patch_click() -> None:
             module._verify_python_env = lambda: None  # type: ignore
 
 
+def shutdown(loop: "asyncio.AbstractEventLoop") -> None:
+    """Cancel all pending tasks on `loop`, wait for them, and close the loop."""
+    from black import concurrency
+
+    return concurrency.shutdown(loop)
+
+
+def cancel(tasks: Iterable["asyncio.Task[Any]"]) -> None:
+    """asyncio signal handler that cancels all `tasks` and reports to stderr."""
+    from black import concurrency
+
+    return concurrency.cancel(tasks)
+
+
+async def schedule_formatting(
```

Comment:
Same thing, is this used anywhere? Right now Black doesn't have an official API (and even then I doubt multiprocessing code should be treated as public).